gen/calc.py

```diff
@@ -759,6 +759,11 @@ def calculate_check_config(check_time):
                     'cmd': ['/opt/mesosphere/bin/dcos-checks', 'ip'],
                     'timeout': '1s'
                 },
+                'docker': {
+                    'description': 'The docker utility is available',
```

Comment:
Let's shorten/clarify the description to `Docker is installed`.

Reply:
Thanks @branden ! I tried to model this based on the existing structure that exist with the other tools. Like how we have it here: https://github.com/bernadinm/dcos/blob/edd057fff898fd4e8b3af7934067902cdd2711b7/gen/calc.py#L737-L750

I can change it from `The docker utility is available` to `Docker is installed` if that is better.

Reply:
I think those descriptions fit better for those checks since they're relatively simple CLI utilities. Docker is a larger beast.

Reply:
Okay got it! Changing it now! Thanks again Branden!